nimbus.fml.yaml

```diff
@@ -202,6 +202,17 @@ features:
         value:
           enabled: true
 
+  total-cookie-protection:
+    description: Enable the total cookie protection to some elegible users.
+    variables:
+      enabled:
+        description: If true, the total cookie protection in all browsing modes.
+        type: Boolean
+        default: false
+    defaults:
+      - channel: developer
+        value:
+          enabled: true
```

Comment:
I think this feature is a bit small.

Would suggest a feature called `web-view-settings` or `web-view`, and have one of the variables called `total-cookie-protection`.

A good test/rule of thumb for what a feature is would be asking "when do you send an exposure event?", or "does the user know when they have been shown the feature?".

Reply:
Thanks for the feedback, I would rather prefer to  `engine-settings` as it's very specific about `WebView` 😅 and we mostly support `GeckoView`.

Reply:
Yes, I added on https://github.com/mozilla-mobile/fenix/pull/25791 because of your feedback :)